router/routes.js

```diff
@@ -2,36 +2,48 @@ require('dotenv').config();
 const mongoose = require('mongoose');
 const express = require('express');
 const questionnaireModel = require('../models/Mongoschema');
-const createQuestionnaire = require('../controller/questionnairecontroller');
+const questionnaireController = require('../controller/questionnairecontroller');
 
 const router = express.Router();
 
-const uri = process.env.DB_URI;
-
 // GET /health --> returns database status
 router.get('/health', function (req, res) {
-  try {
-    mongoose.connect(uri); //TODO: ask for connection status ¿await mongoose.authenticate().then(() =>?
-    res.status(200).set('Content-Type', 'application/json').send({ response: 'Connection has been established successfully.' });
-  } catch (error) {
-    res.status(500).set('Content-Type', 'application/json').send('Unable to connect to the database:');
-  }
+  const dbState = mongoose.STATES[mongoose.connection.readyState];
+    res.set('Content-Type', 'application/json').send({ response: dbState });
 });
 
 // POST /questionnaire --> receives the questionnaire and stores it in the database
```

Comment:
In order to add comments to the endpoints, follow the standard https://apidocjs.com/